packages/cli/test/fixtures/check-npm/terrazzo.config.js

```diff
@@ -1,5 +1,5 @@
 import { defineConfig } from '../../../dist/index.js';
 
 export default defineConfig({
-  tokens: ['my-tokens/tokens.json'],
+  tokens: ['pkg/tokens.json'],
```

Comment:
This is the only thing we’ll have to revert; it’s intentional that it’s using the name in `package.json`, NOT the relative folder name. This is needed to resolve packages in a monorepo or from npm in general. Everything else is great though!